README.md

```diff
@@ -0,0 +1,49 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Portfolio
+
+> This a first mobile project and the project is a fully responsive and a mobile friendly.
+
+
+## Built With
+
+- HTML
+- CSS
+
+## Live Demo (if available)
+
+[Live Demo Link](https://akhror-valiev.github.io/micronauts-portfolio/)
+
+
+
+
+
+## Authors
+
+👤 **Akhror Valiev (Oshie)**
+
+- GitHub: [@akhror-valiev](https://github.com/akhror-valiev)
+- Twitter: [@oshie0115](https://twitter.com/oshie0115)
+- LinkedIn: [@Ahror Valiev](https://www.linkedin.com/in/ahror-valiev-9141911b8/)
```

Comment:
[Optional]
Because it was not mentioned again by @MaganaAsiati, this aspect will be Optional for now, but it may be asked as mandatory by the reviewers in the next milestones (as I explained to you yesterday, missing important sections from the README or the entire README leads to invalidation of the pull request):

- [ ] Please introduce the Getting Started section in the Readme where you give clear explanations to a non-professional user how to replicate by cloning, step by step, your app to their system;